include/UnixStamp.hpp

```diff
@@ -25,29 +25,17 @@
 class UnixStamp
 {
 private:
-    // Both timestamps are saved in GMT+0
+    // The timestamp is in GMT+0
     unixstamp unix;
-    civil_time time;
-    int8_t tz;
 
 public:
     UnixStamp(unixstamp initialUnix);
-    UnixStamp(unixstamp initialUnix, int8_t initialTz);
     UnixStamp(civil_time intialTime);
     UnixStamp(civil_time intialTime, int8_t initialTz);
 
-    civil_time getOriginalTime();
     civil_time getTimeInTz(int8_t tz);
 
     unixstamp getUnix();
-    civil_time getTime();
```

Comment:
`getTime()` and `getOriginalTime()` are removed to emphasize that this class has no notion of a designated timezone.

GMT+0 should not be used as an implicit default, because with other libraries and frameworks the default timezone is often the _local_ timezone as per the system settings. So, not to confuse the user.

The timezone must always be explicitly specified via `getTimeInTz()`.